ql/src/java/org/apache/hadoop/hive/ql/Driver.java

```diff
@@ -2186,6 +2195,43 @@ private TaskRunner launchTask(Task<? extends Serializable> tsk, String queryId,
     return tskRun;
   }
 
+  private void enforceQueue(HiveConf conf) {
+    String queueEnforcerClassName = conf.getVar(HiveConf.ConfVars.QUEUE_ENFORCER_CLASS);
+    QueueEnforcer queueEnforcer = null;
```

Comment:
We should have this initialized at init or something and reuse the same QueueEnforcer object.